lib/web_ui/lib/src/engine/semantics/scrollable.dart

```diff
@@ -121,10 +123,11 @@ class Scrollable extends PrimaryRoleManager {
       _gestureModeListener = (_) {
         _gestureModeDidChange();
       };
-      semanticsObject.owner.addGestureModeListener(_gestureModeListener);
+      EngineSemantics.instance.addGestureModeListener(_gestureModeListener!);
 
+      final Zone zone = Zone.current;
       _scrollListener = createDomEventListener((_) {
-        _recomputeScrollPosition();
+        zone.run(_recomputeScrollPosition);
```

Comment:
Is this necessary? If so, could you add a comment explaining why?

Reply:
Good catch! This was left-over from my experimentation. Will clean up.